.gitignore

```
@@ -1,3 +1,9 @@
-node_modules
+
+**/target
+**/dist
+**/bin
+**/node_modules/
+**/.vscode-test/
+
 *.vsix
 out
```

Maybe add a newline at the end.
```suggestion
out

```